forest/jasmine/mobmat2traj.py

```diff
@@ -44,13 +44,18 @@ def locate_home(MobMat,tz_str):
     Args: MobMat, a k*7 2d array, output from InferMobMat()
           tz_str, timezone, string
     Return: home_x, home_y, two scalar, represent the latitude and longtitude of user's home
+    Raises:
+        AssertionError: if not enough data to infer home location
+
     """
     ObsTraj = MobMat[MobMat[:,0]==2,:]
     hours = []
     for i in range(ObsTraj.shape[0]):
         time_list = stamp2datetime((ObsTraj[i,3]+ObsTraj[i,6])/2,tz_str)
         hours.append(time_list[3])
     hours = np.array(hours)
+    assert ((hours>=19)+(hours<=9)).sum()>0, """No home location found: 
+        Too few observations at night"""
```

Comment:
Any particular reason to choose `assert` vs `RuntimeError` or similar?

Reply:
No particular reason, I just thought it was the most suitable error type for this problem. But I can change it to another type if you think there is one more suitable.